train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -43,7 +46,11 @@ private void enforceSpeedLimit() {
 
 	@Override
 	public void setJoystickPosition(int joystickPosition) {
-		this.step = joystickPosition;		
+		this.step = joystickPosition;
+		TimerTask tt = new TimerTask() {
+			@Override
+			public void run() {followSpeed();}
```

Comment:
Please make sure the format of the run method is consistent with the other methods.

Reply:
Format changed so method formats are consistent.